Kitodo/src/main/java/de/sub/goobi/forms/ProzessverwaltungForm.java

```diff
@@ -991,11 +977,11 @@ public void exportDMSHits() {
             try {
                 export.startExport(proz);
             } catch (Exception e) {
-                Helper.setFehlerMeldung("ExportError", e.getMessage());
-                logger.error(e);
+                Helper.setErrorMessage("ExportError", logger, e);
             }
         }
         Helper.setMeldung(null, "ExportFinished", "");
+        logger.info(Helper.getTranslation("ExportFinished"));
```

Comment:
If you add a log message it should be logged before the user get the information. Please move this log call in front of the `Helper.setMeldung()` call.